code/segmentedtickerplant/datastripe.q

```diff
@@ -22,8 +22,11 @@ jsonchecks:{[]
      //Enable datastriping if all checks pass
      .ds.datastripe:min (0=count errors; 0=count wrongtables);
      $[.ds.datastripe;
-      .lg.o[`jsonchecks;"config checks complete and datastriping is on"];
-      .lg.e[`jsonchecks;"Datastriping is not enabled due to failed config checks"]
+	[.lg.o[`jsonchecks;"config checks complete and datastriping is on"];
```

Comment:
Indentation in this addition seems quite messy and a bit inconsistent

Reply:
I changed it to use tabs but most of the code uses spaces i believe. I should probably just keep it consistent.

Reply:
some lines here still using tabs, others spaces